src/mapping/config/MappingConfiguration.cpp

```diff
@@ -347,6 +350,10 @@ MappingConfiguration::ConfiguredMapping MappingConfiguration::createMapping(
       configuredMapping.mapping = PtrMapping(
           new RadialBasisFctMapping<VolumeSplines>(constraintValue, dimensions, VolumeSplines(), xDead, yDead, zDead));
     } else if (type == VALUE_RBF_GAUSSIAN) {
+      if (!std::isinf(supportRadius)) {
+        // Compute shape parameter from the support radius
+        shapeParameter = std::sqrt(-std::log(Gaussian::cutoffThreshold)) / supportRadius;
+      }
```

Comment:
I could have shifted this over into the `Gaussian` class. However, I decided against this as the class itself can (in theory) still get two independent values (shape-parameter and support-radius).

Reply:
I would also handle it in the configuration.